vagrant/bootstrap.sh

```diff
@@ -44,18 +47,20 @@ parallel --link wget -q -O ::: \
     ${QUORUM_OUTPUT_FILE} \
     ${POROSITY_OUTPUT_FILE} \
     ${CAKESHOP_OUTPUT_FILE} \
+    ${SOLC_OUTPUT_FILE} \
     ::: \
     https://github.com/jpmorganchase/constellation/releases/download/v$CVER/$CREL.tar.xz \
     https://oss.sonatype.org/content/groups/public/com/jpmorgan/quorum/tessera-app/${TESSERA_VERSION}/tessera-app-${TESSERA_VERSION}-app.jar \
     https://oss.sonatype.org/content/groups/public/com/jpmorgan/quorum/enclave-jaxrs/${TESSERA_VERSION}/enclave-jaxrs-${TESSERA_VERSION}-server.jar \
     https://dl.bintray.com/quorumengineering/quorum/v${QUORUM_VERSION}/geth_v${QUORUM_VERSION}_linux_amd64.tar.gz \
     https://github.com/jpmorganchase/quorum/releases/download/v1.2.0/porosity \
-    https://github.com/jpmorganchase/cakeshop/releases/download/v${CAKESHOP_VERSION}/cakeshop-${CAKESHOP_VERSION}.war
+    https://github.com/jpmorganchase/cakeshop/releases/download/v${CAKESHOP_VERSION}/cakeshop-${CAKESHOP_VERSION}.war \
+    https://github.com/ethereum/solidity/releases/download/v${SOLC_VER}/solc-static-linux
 
 # install constellation
 echo "Installing Constellation ${CVER}"
 tar xfJ ${CONSTELLATION_OUTPUT_FILE}
-cp ${CREL}/constellation-node /usr/local/bin && chmod 0755 /usr/local/bin/constellation-node
+cp ${CREL}/constellation-node /usr/local/bin && chmod 0755 /usr/local/bin/constellation-node && chmod 0755 /usr/local/bin/solc
```

Comment:
I don't think the chmod for solc should be here; there's a risk of it getting accidentally removed if constellation is removed.
Suggest moving this to a separate line with an 'install solc' comment (similar to what is done for porosity).

Reply:
Agree. Have done the changes and pushed.